devnet-sdk/testing/testlib/validators/forks.go

```diff
@@ -64,6 +64,8 @@ func IsForkActivated(c *params.ChainConfig, forkName rollup.ForkName, timestamp
 		return c.IsOptimismIsthmus(timestamp), nil
 	case rollup.Interop:
 		return c.IsInterop(timestamp), nil
+	case rollup.Jovian:
```

Comment:
From my earlier comment, should feature flags be added here too? (i.e., `rollup.MinBaseFee`)

My assumption is based on `Interop` is using the feature flag approach